src/lib/button/index.ts

```diff
@@ -1 +1,55 @@
+import {ModuleWithProviders, NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {MdRippleModule} from '../core/ripple/ripple';
```

Comment:
This is failing presubmit because it can't find `MdRippleModule`. Previously we were importing from `../core` rather than using the deep import. Can you change it back?